packages/tooltip/README.md

```diff
@@ -0,0 +1,77 @@
+# Tooltip
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/tooltip.svg)
+
+## Example
+
+<!-- will update once component is finalized -->
+
+## Properties
+
+### open
+
+**Type:** `boolean`
+
+**Default:** `false`
+
+Controls the component, and determines whether or not the `Tooltip` will appear open or closed.
+
+### setOpen
+
+**Type:** `function`
+
+If controlling the component, pass state handling function to setOpen prop. This will keep the consuming application's state in-sync with LeafyGreen's state, while the `Tooltip` component responds to events such as backdrop clicks and a user pressing the Escape key.
+
+### shouldClose
+
+**Type:** `function`
+
+**Default:** `() => true`
+
+Callback to determine whether or not `Tooltip` should close when user tries to close it.
+
+### align
+
+**Type:** `top`, `bottom`, `left`, `right`
+
+**Default:** `top`
+
+Determines the alignment of the `Tooltip` component relative to the element passed to the `trigger` prop.
```

Comment:
Should include that it will be relative to the parent if no trigger is passed.

Reply:
👍